<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>463</width>
         <height>264</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="picLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuLanguage_2">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionGerman"/>
    </widget>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="separator"/>
    <addaction name="action100_Zoom"/>
    <addaction name="actionFit_to_Window"/>
    <addaction name="separator"/>
    <addaction name="actionRGB_YUVedit"/>
    <addaction name="separator"/>
    <addaction name="menuLanguage_2"/>
    <addaction name="actionActionFont"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="action100_Zoom"/>
   <addaction name="actionFit_to_Window"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRGB_YUVedit"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEnglish"/>
   <addaction name="actionGerman"/>
   <addaction name="actionDark_Mode"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>:/Editor Bild/new.png</normaloff>:/Editor Bild/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/open.png</normaloff>:/imgs/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/save-all.png</normaloff>:/imgs/save-all.png</iconset>
   </property>
   <property name="text">
    <string>Save&amp; as</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/print.png</normaloff>:/imgs/print.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/copy.png</normaloff>:/imgs/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/paste.jpg</normaloff>:/imgs/paste.jpg</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>:/Editor Bild/cut.png</normaloff>:/Editor Bild/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/zoom_in.png</normaloff>:/imgs/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom&amp; in</string>
   </property>
   <property name="toolTip">
    <string>Zoom &amp; in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/zoom_out.png</normaloff>:/imgs/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp; out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action100_Zoom">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/zoom_100.png</normaloff>:/imgs/zoom_100.png</iconset>
   </property>
   <property name="text">
    <string> Zoom &amp; 100%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionFit_to_Window">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/zoom_fit.png</normaloff>:/imgs/zoom_fit.png</iconset>
   </property>
   <property name="text">
    <string>Fit to &amp; Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/Image/Button-Help-icon.png</normaloff>:/Image/Button-Help-icon.png</iconset>
   </property>
   <property name="text">
    <string>About Owner</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionDark_Mode">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/darkmode.png</normaloff>:/imgs/darkmode.png</iconset>
   </property>
   <property name="text">
    <string>Dark Mode</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/en.png</normaloff>:/imgs/en.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionGerman">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/de.png</normaloff>:/imgs/de.png</iconset>
   </property>
   <property name="text">
    <string>German</string>
   </property>
  </action>
  <action name="actionGerman_2">
   <property name="text">
    <string>German</string>
   </property>
  </action>
  <action name="actionEnglish_2">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/exit.png</normaloff>:/imgs/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/save.png</normaloff>:/imgs/save.png</iconset>
   </property>
   <property name="text">
    <string>Save&amp;</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionRGB_YUVedit">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/imgs/rgb.png</normaloff>:/imgs/rgb.png</iconset>
   </property>
   <property name="text">
    <string>RGB_YUVedit</string>
   </property>
   <property name="iconText">
    <string>RGB_YUVedit</string>
   </property>
   <property name="toolTip">
    <string>RGB_YUVedit</string>
   </property>
  </action>
  <action name="actionActionFont">
   <property name="text">
    <string>Font</string>
   </property>
   <property name="toolTip">
    <string>Font</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
